lib/Backend/ServerScriptContext.cpp

```diff
@@ -335,13 +335,12 @@ ServerScriptContext::GetEmitBufferManager(bool asmJsManager)
 IR::JnHelperMethod
 ServerScriptContext::GetDOMFastPathHelper(intptr_t funcInfoAddr)
 {
-    IR::JnHelperMethod helper;
+    IR::JnHelperMethod helper = IR::HelperInvalid;
```

Comment:
>HelperInvalid [](start = 36, length = 13)

should we change ScriptContext::GetDOMFastPathHelper accordingly as well?  I'm thinking in-proc jit here, unless the jit always refers to the ServerScriptContext now irrespective of oop or in proc.